### PR TITLE
Changes for Prof. William, Prof. Abdou, and Prof. Kanstantin 

### PR DESCRIPTION
**Pull request by quettabit**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -3592,6 +3593,7 @@ Edward Lank,University of Waterloo,https://cs.uwaterloo.ca/~lank,e7P8IwgAAAAJ
 Edward M. Reingold,Illinois Institute of Technology,https://science.iit.edu/people/faculty/edward-reingold,UlQeH7AAAAAJ
 Edward Omiecinski,Georgia Institute of Technology,http://www.cc.gatech.edu/~edwardo,NOSCHOLARPAGE
 Edward Sciore,Boston College,http://www.cs.bc.edu/~sciore,NOSCHOLARPAGE
+Edward Thomas Grefenstette,University College London,http://egrefen.com/,ezllEwMAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Please point to some source to back up the claim that he is not still at Deep Mind (which is what his home page states).

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 1 above)
Comment (quettabit) [reply csrankings.csv]: @emeryberger https://twitter.com/egrefen/status/1077976894203662337

Comment (quettabit) [reply csrankings.csv]: @emeryberger [He joins FAIR](https://twitter.com/egrefen/status/1082264124543913984 ).

Comment (quettabit) [reply csrankings.csv]: @emeryberger fixed appropriately.

File: csrankings.csv [hunk 2]
```diff
@@ -3860,6 +3861,7 @@ Edward M. Reingold,Illinois Institute of Technology,https://science.iit.edu/peop
 Edward Omiecinski,Georgia Institute of Technology,http://www.cc.gatech.edu/~edwardo,NOSCHOLARPAGE
 Edward P. Tighe,De La Salle University,https://www.dlsu.edu.ph/colleges/ccs/faculty-profile/single/?id=32665809537&command=GETPROFILE,_W85mMIAAAAJ
 Edward Sciore,Boston College,http://www.cs.bc.edu/~sciore,NOSCHOLARPAGE
+Edward Thomas Grefenstette,University College London,http://egrefen.com/,ezllEwMAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: web page says he is at Deep Mind

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 2 above)
Comment (quettabit) [reply csrankings.csv]: Yeah. He is actually at FAIR now. I failed to remove the alias entry earlier.

Comment (quettabit) [reply csrankings.csv]: @emeryberger I guess the PR is good to go now.

**Pull request merged**